WTWSMS/events/religious_events.txt

```diff
@@ -30,7 +30,15 @@ province_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 9600
+		months = 8000
+		modifier = {
+			factor = 0.15
+			AND = {
+				NOT = { year = 500 }
+				religion = nicene # In the early parts of this Mod, the religious world was in turmoil with the fall of Christian WRE
```

Comment:
It was totally ahistorical, though. The WRE was Nicene, and except for the fluke of the Miaphysite ruler that Zeno deposed, so was the ERE in this time period. Nicene should get the penalty, not Miaphysitism. It makes no sense that just because the ERE's ruler is Miaphysite that the Nicene Religion isn't pummeled.